python/azureml/yml/env.yml

```diff
@@ -0,0 +1,23 @@
+# Conda environment specification. The dependencies defined in this file will
+# be automatically provisioned for runs with userManagedDependencies=False.
+
+# Details about the Conda environment file format:
+# https://conda.io/docs/user-guide/tasks/manage-environments.html#create-env-file-manually
+
+name: project_environment
+dependencies:
+  # The python interpreter version.
+  # Currently Azure ML only supports 3.5.2 and later.
+- python=3.6.2
```

Comment:
latest onnxruntime package does not support python 3.6. I recommend python 3.7 or 3.8.
But it is optional since older version of onnxruntime already has optimizations for BERT model.